drizzlepac/devutils/comparison_tools/read_hla/read_hla_image.py

```diff
@@ -0,0 +1,59 @@
+"""read_hla_image.py: read an HLA image (from a file or from HLA webservice) and correct WCS
+
+R. White, 2019 June 20
+"""
+
+from __future__ import print_function
+import omegaxyz, os
+from astropy.io import fits
+
+def read_hla_image(dataset, applyomega=True, verbose=False,
+        url="https://hla.stsci.edu/cgi-bin/getdata.cgi"):
+
+    """Return a astropy.io.fits HDUlist for the given HLA dataset.
+    
+    If dataset is a filename, the file is opened; if dataset is a
+    HLA datasetname, the file is retrieved from the HLA web server.
+    If applyomega is true, the HSC astrometry correction is applied to the WCS in the header.
+    """
+
+    if not os.path.exists(dataset):
+        dataloc = "{}?dataset={}".format(url,dataset)
+    else:
+        dataloc = dataset
+    if applyomega:
+        fh = omegaxyz.updatefits(dataloc, verbose=verbose)
+    else:
+        fh = fits.open(dataloc)
+    return fh
```

Comment:
Same as above comment re: Pep8.

Reply:
Issue resolved in commit [c060957](https://github.com/spacetelescope/drizzlepac/pull/548/commits/c06095740dcc1e7aa3fef2c7a535cb2b03acbfc3)